table-store/src/integration-test/java/com/scalar/dl/tablestore/TableStoreEndToEndTest.java

```diff
@@ -166,37 +163,18 @@ private void insertPaymentRecord(
     clientService.executeStatement(
         "INSERT INTO "
             + PAYMENT_TABLE
-            + " VALUES "
-            + toPartiQLJsonString(createPaymentJson(paymentId, sender, receiver, amount, date)));
+            + " VALUES `"
+            + jacksonSerDe.serialize(createPaymentJson(paymentId, sender, receiver, amount, date))
+            + "`");
   }
 
   private void insertPaymentRecordWithNullValues(int paymentId) {
     clientService.executeStatement(
         "INSERT INTO "
             + PAYMENT_TABLE
-            + " VALUES "
-            + toPartiQLJsonString(createPaymentJson(paymentId)));
-  }
-
-  private String toPartiQLJsonString(JsonNode node) {
```

Comment:
This is no longer necessary since we can write the native JSON with the backquotes.